### DISCUSSION TITLE
Separate mol object with several molecules

### DISCUSSION THREAD

**Discussion by OphirKa** [rdkit/rdkit#5004] [Q&A] [answered]
I want to write algorithm that receive mol object which contains at least 2 molecules and separate the to 2 mol objects (or any other way that would help me to refer each one separately.
(I converted those mol files from cif files through openbabel, and the cartesian coordinates need to stay as is)
Is there any exist option in rdkit?

Thank you

**Comment by bp-kelley** [comment, marked as answer]
Look for GetMolFrags on this page:

https://www.rdkit.org/docs/source/rdkit.Chem.rdmolops.html

----
Brian Kelley
